packages/web-components/src/message-bar/message-bar.ts

```diff
@@ -0,0 +1,70 @@
+import { attr, FASTElement } from '@microsoft/fast-element';
+import { MessageBarIntent, MessageBarLayout, MessageBarPoliteness, MessageBarShape } from './message-bar.options.js';
+
+/**
+ * A Message Bar Custom HTML Element.
+ *
+ * @slot actions - Content that can be provided for the actions
+ * @slot close - Content that can be provided for the close button
+ * @slot - The default slot for the content
+ * @public
+ */
+export class MessageBar extends FASTElement {
+  /**
+   * Sets the aria-labelledby attribute of the control.
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: `aria-labelledby`
+   */
+  @attr({ attribute: 'aria-labelledby' })
+  ariaLabelledBy: string | null = null;
```

Comment:
I wouldn't observe any `aria-` attributes here as those come for free from the platform.

Reply:
Can I just remove this, or do I need to setup anything else in the class to enable this?

Reply:
This can just be removed, it can be set by implementors with:
```html
<message-bar aria-labelledby="foo"></message-bar>
```